File: python/uri/uri-1008.py
employer_id = int(input())
employer_total = int(input())
employer_salary_per_hour = float(input())
employer = [employer_id,employer_total,employer_salary_per_hour]
employer_salary = employer[1]*employer[2]
print("NUMBER = %d\nSALARY = U$ %.2f" % (employer[0],employer_salary))

File: c/uri/uri1008.c
#include <stdio.h>

void main(){
    int employer_id, employer_worked_hours;
    double employer_hour_amount;

    scanf("%d", &employer_id);
    scanf("%d", &employer_worked_hours);
    scanf("%lf", &employer_hour_amount);

    double employer_salary = employer_hour_amount * employer_worked_hours;

    printf("NUMBER = %d\n", employer_id);
    printf("SALARY = U$ %0.2lf\n", employer_salary);
}